bundles/SystemInfoBundle/src/SystemInfoBundle.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Pimcore\Bundle\SystemInfoBundle;
+
+use Pimcore\Bundle\SystemInfoBundle\DependencyInjection\SystemInfoExtension;
+use Pimcore\Extension\Bundle\AbstractPimcoreBundle;
+use Pimcore\Extension\Bundle\Traits\PackageVersionTrait;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+
+class SystemInfoBundle extends AbstractPimcoreBundle
```

Comment:
Please adapt the bundle naming according to [this](https://github.com/pimcore/pimcore/issues/13912), for example
```suggestion
class PimcoreSystemInfoBundle extends AbstractPimcoreBundle
```
Also rename the file